### PR TITLE
Add Cooking Mama: Sweet Shop

### PR DESCRIPTION
**Pull request by HardWitchMismay**
Adds Cooking Mama: Sweet Shop.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/cooking-mama-sweet-shop/game.dat [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+[[releases]]
+title = "00040000001B9C00"
+region = "usa"
+release_date = "2017-05-18"
+
+[[testcases]]
+title = "00040000001B9C00"
+compatibility = "1"
+date = "2017-12-18"
+version = "HEAD-040006f"
```
Comment (TheKoopaKingdom) [inline games/cooking-mama-sweet-shop/game.dat]: This commit is 11 days old, please retest on the latest nightly.

File: games/cooking-mama-sweet-shop/game.dat [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+cpu = "Intel Celeron 1007U @ 1.50GHz"
```
Comment (TheKoopaKingdom) [inline games/cooking-mama-sweet-shop/game.dat]: The `@ 1.50GHz` isn't needed.

File: games/cooking-mama-sweet-shop/game.dat [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+gpu = "Intel HD Graphics"
+os = "Windows 10 16299.125"
```
Comment (TheKoopaKingdom) [inline games/cooking-mama-sweet-shop/game.dat]: The `.125` isn't needed.

File: games/cooking-mama-sweet-shop/game.dat [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+region = "usa"
+release_date = "2017-05-18"
+
+[[testcases]]
+title = "00040000001B9C00"
+compatibility = "1"
+date = "2017-12-18"
+version = "HEAD-040006f"
+cpu = "Intel Celeron 1007U @ 1.50GHz"
+gpu = "Intel HD Graphics"
```
Comment (TheKoopaKingdom) [inline games/cooking-mama-sweet-shop/game.dat]: Not specific enough.

**Comment by Hexagon12** [comment]
Please adress the feedback or this PR will be closed in due time.

**Comment by Hexagon12** [comment]
Closed due to inactivity.

**Pull request closed without merging**